.github/workflows/platform.dependencies.yml

```diff
@@ -23,6 +23,7 @@ on:
   #     - 'utilities/pipelines/dependencies/**'
 
 env:
```

Comment:
Did you run the pipeline end to end?

Reply:
yes. But need to run another set of tests after some major PRs being merged. Will update status